jest.config.js

```diff
@@ -11,11 +11,7 @@ module.exports = {
   // Ignore the tests that are still using tap/tape for as their test runner
   testPathIgnorePatterns: [
     `./packages/cactus-plugin-keychain-aws-sm/src/test/typescript/integration/openapi/openapi-validation.test.ts`,
```

Comment:
@AndreAugusto11 You also have to make these same ignore/include pattern changes in the .taprc file so that the tests that are now using Jest will not be executed by tap/tape. (so head over to that file and remove these files from the include list is what you need to do if I remember correctly).

Once that's sorted out the CI should start passing, assuming that the tests are all working fine otherwise.

Reply:
@petermetz that was what I was missing! I didn't know about that config file. It is now passing all checks. TY